vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token.cc

```diff
@@ -61,10 +61,12 @@ void RedeemToken::Redeem(
     return;
   }
 
-  auto token_info = unblinded_tokens_->GetToken();
-  unblinded_tokens_->RemoveToken(token_info);
+  const TokenInfo token = unblinded_tokens_->GetToken();
+  unblinded_tokens_->RemoveToken(token);
 
-  CreateConfirmation(info, confirmation_type, token_info);
+  const ConfirmationInfo confirmation =
+      CreateConfirmation(ad, confirmation_type, token);
+  CreateConfirmation(confirmation);
```

Comment:
Can the first variation of `CreateConfirmation` just call the other one internally? It looks a little odd to call `CreateConfirmation` twice in a row.

Reply:
Unfortunately not as the first variation is used elsewhere and does not need to call this function

Reply:
Perfect

Reply:
Ideally, I think these two functions would have different names, since they appear to have different responsibilities.

Reply:
`CreateConfirmation` creates a confirmation on the backend, `CreateConfirmationInfo` creates a data structure for a confirmation